packages/smart-contracts/scripts/deploy-one.ts

```diff
@@ -0,0 +1,37 @@
+import { Contract } from 'ethers';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+import { ContractArtifact } from '../src/lib';
+
+export default async function deploy(
```

Comment:
```suggestion
export default async function deployOne(
```

btw, shouldn't it be `deployOnce`? as it ensures only 1 instance is delpoyed?

Reply:
I took the habit to name them all `deploy()` and to have it the only function exported by each script, as default. So the name is quite optional actually... Is it bad practice?

Reply:
As it is imported as `deployOne`, I would name it `deployOne`. Easier for searches and, even more, autocomplete. I tend to avoid `export default function` and prefer `export function`, I find it to be much more powerful for autocomplete. 

(yes, this codebase is full of `export default`, I know ^^)